lm_eval/tasks/superglue.py

```diff
@@ -69,6 +69,9 @@ def has_validation_docs(self):
     def has_test_docs(self):
         return True
 
+    def fewshot_description(self):
+        return "Given a premise and a hypothesis, classify whether the author of the premise is committed to the truth of the hypothesis. The three possible labels are true, false or neither."
+
     def doc_to_text(self, doc, include_target=True):
         text = "{}\nquestion:\t{}\ttrue, false or neither?\nanswer:".format(
```

Comment:
Should it be `\t` or just a regular space? I personally think just a regular space makes more sense but we might want to check with GPT3 authors

Reply:
Replaced tabs with spaces to keep things consistent until we decide to reinstate for all cases.